jigna/js/build.js

```diff
@@ -68,3 +69,37 @@ concatenate({
         'jigna'
     ]
 });
+
+// jigna-vue.js
+concatenate({
```

Comment:
`concatenate` shouldn't be called twice. Maybe for now we simply ship both angular and vue in the distributed jigna.js keeping the choice of the backend up to the user but later we can make it a build option to serve a `jigna.angular.js` and a `jigna.vue.js` separately.